tools/java/org/unicode/cldr/util/data/languages/isolates.json

```diff
@@ -0,0 +1,3 @@
+[
+	"ko", "qu", "root"
```

Comment:
Question: what is this for?

Reply:
Delayed answer:  it's to move `static final Set<String> ISOLATES` out to a data file.